style.css

```diff
@@ -1,97 +1,396 @@
   font-style: normal;
   font-weight: 400;
   line-height: 64px;
+  letter-spacing: 0.3700000047683716px;
   text-align: left;
-  color: #ff6b00;
+  margin-top: 120px;
 }
 
 h2 {
-  width: 324px;
-  height: 27px;
-  margin: 0 24px 0 24px;
+  color: #fff5e1;
   font-family: "Roboto", sans-serif;
   font-size: 24px;
   font-style: normal;
   font-weight: 700;
   line-height: 32px;
+  letter-spacing: 0;
   text-align: left;
-  color: #fff;
 }
 
 p {
-  height: 123px;
-  margin: 12px 24px 0 24px;
   font-family: "Poppins", sans-serif;
   font-size: 16px;
   font-style: normal;
   font-weight: 400;
   line-height: 24px;
   letter-spacing: 0;
   text-align: left;
-  color: #fff;
+  color: white;
 }
 
-ul {
+.social-media {
   display: flex;
-  justify-content: space-between;
-  align-items: center;
-  width: 141px;
+  gap: 22.3px;
+  margin-bottom: 130px;
 }
 
-li {
+.social-media > li {
   list-style-type: none;
 }
 
-.nav-menu {
+.work-section {
+  display: grid;
+  grid-template-columns: 100%;
+}
+
+.work-section > * {
+  padding: 0 24px 0 24px;
+}
+
+.work-1 {
   display: flex;
-  justify-content: space-between;
-  height: 50px;
-  width: 375px;
-  left: 0;
-  top: 0;
-  border-radius: 0;
-  background-color: #3c3a39;
-  align-items: center;
+  flex-direction: column;
 }
 
-.logo {
-  width: 77px;
-  height: 24px;
-  margin-left: 12px;
-  margin-top: 10px;
-  font-family: "Roboto", Times, serif;
-  font-size: 18px;
+.work1-visual {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work1-visual > h1 {
+  font-family: "Crete Round", serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #1c1a19;
+}
+
+.line {
+  border: 1px solid black;
+  margin: 0 24px 56px 24px;
+}
+
+.work1-content {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.work1-content > h3 {
+  margin-top: 24px;
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #091e42;
+}
+
+.work1-content > p {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #091e42;
+}
+
+.work1-content > ul {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 12px;
+  list-style: none;
+}
+
+.list1 {
+  padding: 12px;
+  text-align-last: center;
+  border: 1px solid black;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #091e42;
+}
+
+.work1-button {
+  padding-top: 10px;
+  background-color: #ff6b00;
+  grid-column: 1 / span 2;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
   font-style: normal;
   font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
```

Comment:
- Kindly make sure to align the image width with the button width as per the figma design.
Your design:
<img width="366" alt="Screenshot 2022-03-24 at 9 11 59 PM" src="https://user-images.githubusercontent.com/70269507/159987037-4e7d4e8d-bf07-4ef5-86e5-e70aafdacb1a.png">

Figma design:
<img width="306" alt="Screenshot 2022-03-24 at 9 11 42 PM" src="https://user-images.githubusercontent.com/70269507/159987141-7d3af07e-ead7-4e96-b372-c8747917f461.png">